plugins/org.wso2.developerstudio.eclipse.ds.editor/DSSEditor/assets/js/data-handler.js

```diff
@@ -214,15 +214,15 @@ $(document).ready(function ($) {
         	return false;
         }
         
-        if (username == "") {
-        	showDSNotification("danger", "Please provide the username of the connection.", 6000);
-        	return false;
-        } 
-        
-        if (password == "") {
-        	showDSNotification("danger", "Please provide the password of the connection.", 6000);
-        	return false;
-        }
+//        if (username == "") {
+//        	showDSNotification("danger", "Please provide the username of the connection.", 6000);
+//        	return false;
+//        } 
+//        
+//        if (password == "") {
+//        	showDSNotification("danger", "Please provide the password of the connection.", 6000);
+//        	return false;
+//        }
```

Comment:
Added a comment here because Derby connection does not need username or password to create a connection. If only the Derby database connection behaves like that, we can add a condition to skip the username and password empty validation for Derby data sources.